integration/mapreplay.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
+. "${INTEGRATION_DIR}"/functions.sh
+
+MAP_JOURNAL=${1}
+if [[ ! -f "${MAP_JOURNAL}" ]]; then
+  echo "First argument should be map journal file"
+  exit 1
+fi
+
+declare -a OLD_MAP_ARRAY
```

Comment:
Would it be better to have the previous run output the map configs? This would make parsing simple and correct.

Reply:
At the moment, the shell script does the tree provisioning, so there's no `CreateTree` in the journal to replay.  

(Shifting things so that the maphammer creates its own tree, as happens in other places, would probably make sense, but is a bit beyond the scope of this PR.)